go/Makefile

```diff
@@ -1,12 +1,18 @@
 COINSTACKS := $(wildcard coinstacks/*)
 MAKEFLAGS += --no-print-directory
 
-.PHONY: all build clean $(COINSTACKS)
+.PHONY: all build clean $(COINSTACKS) husky
```

Comment:
we shouldn't need to add husky here. it is auto installed in the prepare hook when running `yarn` in `unchained/node`.

the readme should probably be updated to require `yarn` in `unchained/node` as a setup step to ensure all of the linting dependencies are set up and installed.